CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Pending
 
+## v0.3.0
```

Comment:
@ValarDragon - not sure I follow these versions.
The current tags of the repo are: `0.9.0` and `1.0.0`. If we consider the project to still be under active development, we can follow item 4 of the semver: https://semver.org/#spec-item-4 that says that the major can still be kept as `0` for initial development - indicating that the public API should not be considered stable. That comes at handy if we do not want to start maintaining majors at this stage (i.e. no stability/backward compatibility guarantees at this stage).
Hence, it seems to me that our current tags: `0.9.0` and `1.0.0` need to become `0.9.0` and `0.10.0`. However, since no tags precede `0.9.0` we can simply reset to `0.1.0`, meaning that:
- `0.9.0` is effectively changed to `0.1.0`
- `1.0.0` is changed to `0.2.0`
- And thus this PR becomes `0.3.0`

What do you think?

Reply:
Sorry, just realized I forgot to reply to this message. 

I am of the same opinion regarding how to handle the versions. (There 0.3.0 refers to this version)
I should have remove the ## Pending for the merge to master, and just add that back onto develop after the PR is merged.

Reply:
Alright, perfect. I changed the name of this PR in line with the comment above. We now need to adjust the existing tags to match the versions listed above, and upon merge of this PR we will be tag the merge commit with `0.3` then.

Reply:
Tags changed.